src/main.js

```diff
@@ -32,19 +31,19 @@
   var ffts = initializeFFTs(20, bufferSize);
   var buffer = null;
 
-  var renderer = new THREE.WebGLRenderer();
-  renderer.setPixelRatio(window.devicePixelRatio ? window.devicePixelRatio : 1);
-  renderer.setSize(resolution * aspectRatio, resolution);
-  renderer.domElement.style.width = '100%';
-  renderer.domElement.style.height = 'auto';
-  // window.addEventListener('resize', function(){
-  //   let canvasWidth = document.querySelector('.col').offsetWidth;
-  //   resolution = canvasWidth/16*10;
-  //   renderer.setSize(resolution * aspectRatio, resolution);
-  //   renderer.domElement.height = resolution * window.devicePixelRatio;
-  //   renderer.domElement.width = canvasWidth * window.devicePixelRatio;
-  // });
-  document.querySelector('#showcase').appendChild(renderer.domElement);
+  var renderer = new THREE.WebGLRenderer({ canvas: document.querySelector("canvas") });
+
+
+  function resize() {
+    var resolution = renderer.domElement.parentNode.offsetWidth / 16 * 10;
+    renderer.setPixelRatio(window.devicePixelRatio ? window.devicePixelRatio : 1);
+    renderer.setSize(resolution * aspectRatio, resolution);
+    renderer.domElement.style.width = '100%';
```

Comment:
setSize is required, but I've revised the way I'm setting the canvas size and it looks very crisp now.